homepage/views.py

```diff
@@ -53,3 +55,23 @@ def add_review(request):
     else:
         form = ReviewForm()
     return render(request, 'homepage/add_review.html', {'form': form})
+
+
+def sign_up(request):
+    if request.method == "POST":
+        form = SignUpForm(request.POST)
+
+        if form.is_valid():
+            appUser = AppUser()
+            appUser.user = form.save()
+            appUser.save()
+
+            login(request, appUser.user)
+            messages.success(request, "Registration successful.")
+            return redirect("sign_in")
+
+        else:
+            messages.error(request, "Unsuccessful registration. Invalid information.")
```

Comment:
Is there no way to determine which field is invalid, specifically?

Reply:
Same like Vera, a more verbose error message will be user friendly

Reply:
Regarding the comment on the test file - if i remove even if i state this PR is dependent on #99 the automatic tests on github will fail - so i prefer to keep it like that.

Regarding the messages - they are only partly implemented, so this will be improved in a separate PR.

Reply:
If you have an error message already, no matter how basic, that just says "oops, can't do that",
And it's implemented (as it is here)
then pass it on to render and five it the 3 lines in the html file.
Further implementation should include more specific error messages, if we want them,
but I believe that if you're calling upon it and giving it a value, it is ready for display.

Reply:
@VeraTs 
I'm convinced - added the messaged in the HTML file.